packages/editor-ui/src/utils/externalHooks.ts

```diff
@@ -12,7 +11,7 @@ export async function runExternalHook(eventName: string, store: Store, metadata?
 		const hookMethods = window.n8nExternalHooks[resource][operator];
 
 		for (const hookMethod of hookMethods) {
-			await hookMethod(store, metadata);
+			await hookMethod(metadata);
```

Comment:
this would be major breaking change.. hooks could also be being used by embed partners.. could we keep the store there at least? since without it, hooks are somewhat useless